source/content/object-cache.md

```diff
@@ -213,7 +209,7 @@ This configuration uses the `Redis_CacheCompressed` class for better performance
 
 1. Enable the Redis cache server from your Pantheon Site Dashboard by going to **Settings** > **Add Ons** > **Add**. It may take a couple minutes for the Redis server to come online.
 
-1. Add the [Redis](https://www.drupal.org/project/redis) module from Drupal.org. You can install and enable the module from the command line using [Terminus](/terminus):
+1. Add the [Redis](https://www.drupal.org/project/object-cache) module from Drupal.org. You can install and enable the module from the command line using [Terminus](/terminus):
```

Comment:
broken link - the Drupal project is located at https://www.drupal.org/project/redis